server/protocols/webadmin-cli/src/main/java/org/apache/james/cli/domain/DomainCreateCommand.java

```diff
@@ -43,7 +43,8 @@
     @Override
     public Integer call() {
         try {
-            DomainClient domainClient = Feign.builder()
+            DomainClient domainClient = new FeignClientFactory(domainCommand.webAdminCli.jwt)
```

Comment:
Can domainCommand expose methods instead of fields?

Reply:
(domainCommand can have a `.feignClientFactory()` method, so that each and every command don't need to be aware of JWT...